skillmap/src/components/Modal.tsx

```diff
@@ -38,30 +40,29 @@ export class Modal extends React.Component<ModalProps> {
             <div className={`modal ${className || ""}`} onClick={this.handleModalClick} role="dialog">
                 <div className="modal-header">
                     {fullscreen && <div className="close button" onClick={this.handleCloseClick}>
-                        <i className="icon arrow left" />
+                        <i className="fas fa-arrow-left" />
                         {lf("Go Back")}
                     </div>}
                     <div className="modal-title">{title}</div>
                     {!fullscreen && <div className="spacer" />}
-                    {!fullscreen && <div className="modal-close-icon" onClick={this.handleCloseClick} role="button"><i className="icon close remove circle" /></div>}
+                    {!fullscreen && <div className="modal-close-icon" onClick={this.handleCloseClick} role="button"><i className="fas fa-times-circle" /></div>}
                 </div>
                 <div className="modal-body">
                     {this.props.children}
                 </div>
                 {actions && actions.length > 0 && <div className="modal-actions">
-                    {actions.map((el, i) => {
-                        return el.url
-                            ? <a key={i} className={`modal-button ${el.className || ""}`} href={el.url} onClick={el.onClick} target="_blank" rel="noopener noreferrer" role="button">{el.label}</a>
```

Comment:
i think these were the only two places using `modal-button` so we could possibly delete all CSS related to it